src/main/scala/com/romix/akka/serialization/kryo/KryoSerializerExtension.scala

```diff
@@ -68,6 +68,9 @@ object KryoSerialization {
 
     val KryoCustomSerializerInit: String = Try(config.getString("akka.actor.kryo.kryo-custom-serializer-init")).getOrElse(null)
 
+    val KryoDefaultSerializer: String = Try(config.getString("akka.actor.kryo.kryo-default-serializer"))
+      .getOrElse("com.esotericsoftware.kryo.serializers.FieldSerializer")
```

Comment:
I'd rather have the default set in the `reference.conf` with a short doc about it